community/release.md

```diff
@@ -1,17 +1,22 @@
+# Capact release process
+
+This document describes Capact release process.
+
 ---
-sidebar_position: 6
----
+**NOTE**
 
-# Release process
+Releasing new Capact version requires admin privileges in the `capact`, `hub-manifests` and `website` repositories.
 
-This document describes Capact release process.
+---
```

Comment:
1. Please revert the changes to the header:


```markdown
---
sidebar_position: 6
---

# Release process

This document describes Capact release process.
```

2. Also, we can easily convert the NOTE to one of prerequisites ("Prerequisites" section). Let's remove it from here and apply the suggestion below 🙂